data/json/recipes/weapon/ranged.json

```diff
@@ -155,6 +155,72 @@
     "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ] ],
     "components": [ [ [ "2x4", 2 ] ], [ [ "pipe", 1 ] ], [ [ "sharp_rock", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "longrifle_flintlock_revolver",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "fabrication",
+    "skills_required": [ "rifle", 5 ],
+    "difficulty": 10,
+    "time": "560 m",
+    "autolearn": true,
```

Comment:
You can't both autolearn a recipe and it explicitly be a replica of someone else's design.

Reply:
fixed